pkg/config/config_unsupported.go

```diff
@@ -5,3 +5,9 @@ package config
 func selinuxEnabled() bool {
 	return false
 }
+
+// Capabilities returns the capabilities parses the Add and Drop capability
+// list from the default capabilities for the container
+func (c *Config) Capabilities(user string, addCapabilities, dropCapabilities []string) ([]string, error) {
+	return []string{}, nil
```

Comment:
should this just return nil. nil? There seems to be no need to allocate an empty slice here and given no caps are supported nil might be a better return value?

Reply:
That works just as well, thanks!